src/pages/home/report/ReportActionsView.js

```diff
@@ -83,22 +83,22 @@ class ReportActionsView extends React.Component {
         this.state = {
             isLoadingMoreChats: false,
         };
+
+        this.updateSortedReportActions(props.reportActions);
     }
 
     componentDidMount() {
         AppState.addEventListener('change', this.onVisibilityChange);
         subscribeToReportTypingEvents(this.props.reportID);
         this.keyboardEvent = Keyboard.addListener('keyboardDidShow', this.scrollToListBottom);
         this.recordMaxAction();
-        fetchActions(this.props.reportID);
+        fetchActions(this.props.reportID).catch(console.error);
```

Comment:
We should not be catching anything at all here. If we want to catch these errors it should be done in `fetchActions()` but no need to return a promise or handle it here.

Reply:
Sorry It's a habit I have to not leave promises get swallowed. It could sometime hide errors:
![image](https://user-images.githubusercontent.com/12156624/114055443-7547f380-9899-11eb-85dd-fed13bc1c7de.png)

Maybe you could make `fetchActions` return `void` so that it's not used incorrectly